round4/team-06.md

```diff
@@ -0,0 +1,140 @@
+)
+
+<i>(Quick overview screenshots attached)</i>
+
+### API, Libraries, and Frameworks
+
+- [OpenAI] (https://platform.openai.com/docs) : We will use the OpenAI API to query the content we want to summarize.
+- [ReactJS] (https://react.dev) : Utilize ReactJS for the frontend to create components.
+- [Some JS library for web scraping] (TBD): We plan to either find a JS library to help us with web scraping or simply draw from targeted DOM elements. 
+
+### Data Storage
+
+- Utilize local storage to store the user’s API Key and the previous summary/keywords/questions.
+
+## Project Management
+
+### Collaboration and Task Allocation
+
+Leader: Emma Lee
+Manager: Rohan Kamat
+Remaining Team Members: Annie Wang, Kevin Lee
+
+- [Kevin] Repository Setup and establishing Chrome extension foundation
+  - Kevin will spearhead the initial setup of our project repository, which includes configuring the foundation, setting up the basic structure and backbone. 
+- [Emma] User Interface Design
+  - Emma will take charge of designing the user interface, starting with creating the Figma mocks for our extension. Then, she will develop the sidebar features, UI components for the user interface. 
+- [Annie] Web Page Text Extraction 
+  - Annie will be responsible for researching and implementing ways to extract text from webpages. This includes researching the appropriate JavaScript libraries for text extraction, manipulation DOM to access specific text elements, and ultimately implementing text extraction. 
+- [Rohan] OpenAI API Integration and Prompt Engineering 
+  - Rohan will be responsible to initiate the integration with OpenAI API and craft the prompts that will work the best for generating summaries. This includes setting up the API calls within our extension, experimenting with different prompt structures to achieve the best summaries, and ensuring our extension interacts efficiently with the API. 
+
+To ensure effective collaboration and communication throughout the project, we plan on using the following platforms: 
+
+- Github: We will use it for code sharing and allowing everyone to collaborate effectively. 
+- Slack: We will utilize Slack for meeting coordinations, general communications, check-ins, updates, and team discussions. 
+- Google Meets: In addition to in-person meetings, we will be utilizing Google Meets to meet virtually as needed throughout the course of the project. 
+
+### Risks and Mitigation
+
+- Limited time to meet which may cause duplication of work. To prevent this, we will try to meet up during class time to ensure that communication is swift.
+- Having issues with setting up the environment. Because everyone is new with the development of Chrome extensions, there may be some technical issues trying to use ReactJS to create extensions. If there are issues, we can ask the TAs or browse the internet to find solutions or tutorials to set up the environment.
```

Comment:
Yes I think watching Youtube Channels helps a lot. If you can afford it, I will also recommend Udemy. Setting up the environment will be very important and you want to understand this part well for a seamless integration.